project/AppDependencies.scala

```diff
@@ -13,13 +13,13 @@ object AppDependencies {
     ws,
     "uk.gov.hmrc"             %% "bootstrap-frontend-play-30" % bootstrapVersion,
     "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-30"         % mongoVersion,
-    "uk.gov.hmrc"             %% "play-frontend-hmrc-play-30" % "9.11.0",
-    "uk.gov.hmrc"             %% "play-partials-play-30"      % "9.1.0",
-    "uk.gov.hmrc"             %% "domain-play-30"             % "9.0.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc-play-30" % "11.3.0",
```

Comment:
Revert this back to 9.11.0 and it will fix the broken test. The last epic we completed for play-frontend was v9, v10 is a major release so you could have a look at it but this might need to be dealt with in another ticket